src/org/thoughtcrime/securesms/database/MessageReactionDatabase.java

```diff
@@ -22,15 +22,18 @@
     public static final String HASH_ID       = "hash";
     public static final String REACTION_DATE = "date_sent";
 
-    public static final String[] PROJECTION = {MessageReactionDatabase.REACTION,
+    public static final String[] PROJECTION = {
+            MessageReactionDatabase.REACTION,
             MessageReactionDatabase.SMS_HASH,
             MessageReactionDatabase.MMS_HASH,
             MessageReactionDatabase.REACTION,
             MessageReactionDatabase.REACTOR_ID,
             MessageReactionDatabase.REACTION_DATE,
-            MessageReactionDatabase.HASH_ID};
+            MessageReactionDatabase.HASH_ID
+    };
 
-    public static final String CREATE_TABLE = "CREATE TABLE " + TABLE_NAME + "(" + ID + "  INTEGER PRIMARY KEY, " +
+    public static final String CREATE_TABLE = "CREATE TABLE " + TABLE_NAME + "(" +
+            ID + "  INTEGER PRIMARY KEY, " +
```

Comment:
We should make sure that `PRIMARY KEY` does not break the default behavior of the table

Reply:
Isn't this just a formatting change?